src/platform/platform_winuser.c

```diff
@@ -153,6 +153,9 @@ CxPlatProcessorInfoInit(
         Status = QUIC_STATUS_OUT_OF_MEMORY;
         goto Error;
     }
+    CxPlatZeroMemory(
```

Comment:
should this be in `CXPLAT_ALLOC_NONPAGED` ? wouldn't it be best if QUIC could consistently reason about whether a specific allocation is zero'd?

Reply:
In the future, we can definitely look at putting it directly in there or perhaps creating a `CXPLAT_ALLOC_NONPAGED_ZERO` or something. Right now, I just want a spot fix.

Reply:
I believe HeapAlloc has a flag for zero-initializing the memory, so we wouldn't even need to do anything extra but pass the flag.

Reply:
If you change `CxPlatAlloc` for example, you can use `HEAP_ZERO_MEMORY` to do the work as efficiently as possible. `ExAllocatePool2` also has a flag.

Reply:
Yes, it can easily be done. I just want to take some time to figure out the best/right way.

Reply:
At the very least, this needs to apply to winkernel, too.

Reply:
Hmm, I see, this pattern isn't used in winkernel. NVM.